.env.base

```diff
@@ -46,7 +46,8 @@ REACT_APP_FEATURE_FOX_PAGE=false
 REACT_APP_FEATURE_FOX_PAGE_RFOX=false
 REACT_APP_FEATURE_FOX_PAGE_FOX_SECTION=true
 REACT_APP_FEATURE_FOX_PAGE_FOX_FARMING_SECTION=false
-REACT_APP_FEATURE_PHANTOM_WALLET=false
+REACT_APP_FEATURE_PHANTOM_WALLET=true
```

Comment:
Nah I just cherry-picked https://github.com/shapeshift/web/pull/7964
 in here before it went in, since I noticed this flag went away in a wrong mergefix in one of the FOX Page PRs - should look good after a develop merge in a sec

Reply:
7e2a8b56341d91b37c3148e53f9bfdfe5775bec9